dockstore-client/src/test/java/io/dockstore/client/cli/WesFileIT.java

```diff
@@ -0,0 +1,90 @@
+package io.dockstore.client.cli;
+
+import io.dockstore.client.cli.nested.WesFile;
+import io.dropwizard.testing.ResourceHelpers;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.contrib.java.lang.system.ExpectedSystemExit;
+import org.junit.contrib.java.lang.system.SystemErrRule;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+public class WesFileIT {
```

Comment:
You should remove `IT` -- makes it look like an integration test, but it's unit tests. I think the ending `IT` has implications in how tests are run on CircleCI (I forget the details), although it may not matter here, but in any case you should remove `IT`.

Reply:
Fixed: https://github.com/dockstore/dockstore-cli/pull/113/commits/33d49f0d5644fc8602848115643cd8aa08222d72